lib/dnsimple/client/clients.rb

```diff
@@ -57,7 +57,7 @@ def webhooks
     end
 
 
-    # @!class Struct
+    # Struct
```

Comment:
Not sure what this comment (or the one before your modification) means in here. Can you explain?